site/assets/js/Pricing-0.4.0.js

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (c) 2020, Inversoft Inc., All Rights Reserved
+ */
+'use strict';
+
+// noinspection DuplicatedCode
+class FusionAuthPriceCalculator {
+  constructor() {
+
+    this.hostingSelectionDiv = document.getElementById('hosting-selection');
+    this.hostingPriceDiv = document.getElementById('hosting-price');
+    this.planSelectionDiv = document.getElementById('plan-selection');
+    this.planPriceDiv = document.getElementById('plan-price');
+    this.sumDiv = document.getElementById('sum');
+    this.monthlyActiveUserSlider = document.querySelector('input[name=monthly-active-users]');
+    this.monthlyActiveUserSlider.addEventListener('input', event => this._handleSliderChange(event));
+    this.monthlyActiveUserSlider.addEventListener('mouseup', event => this._handleSliderChange(event));
+    this.monthlyActiveUserSliderMin = parseInt(this.monthlyActiveUserSlider.getAttribute('min'));
+    this.monthlyActiveUserSliderMax = parseInt(this.monthlyActiveUserSlider.getAttribute('max'));
+    this.monthlyActiveUserValue = document.getElementById('monthly-active-users-value');
+    this.communityButton = document.querySelector('a[data-plan=Community]');
+    this.starterButton = document.querySelector('a[data-plan=Starter]');
+    this.purchaseButton = document.getElementById('purchase-button');
+
+    this.yearlyRadio = document.getElementById('yearly_radio');
+    this.yearlyRadio.addEventListener('change', event => this._handleBillingIntervalChange(event));
+    this.monthlyRadio = document.getElementById('monthly_radio');
+    this.monthlyRadio.addEventListener('change', event => this._handleBillingIntervalChange(event));
+
+    this.hostingPrice = 0;
+    this.planPrice = 0;
+    this._loadState();
+
+    this.yearlyRadio.checked = true;
+    this.billingInterval = 'yearly';
+
+    document.querySelectorAll('a[data-step]')
+            .forEach(e => e.addEventListener('click', event => this._handleStepClick(event)))
+    document.querySelectorAll('a[data-plan]')
+            .forEach(e => e.addEventListener('click', event => this._handlePlanClick(event)))
+    window.addEventListener('popstate', event => this._handleStateChange(event));
+
+    fetch('https://account-local.fusionauth.io/ajax/purchase/price-model')
+        .then(response => response.json())
+        .then(json => {
+          this.priceModel = json;
+          this._changeStep();
+          this._drawPlanPrices();
+        })
+  }
+
+  _calculateHostingPrice(type) {
```

Comment:
In our other projects (`fusionauth-app` and `account.fusionauth.io`) we've leaned more into the ECMAScript 6 which suggestions using `#` as a prefix for private variables and methods. 

For example, this method would be `#calculateHostingPrice`.